modules/EmailMan/views/view.config.php

```diff
@@ -138,6 +138,7 @@ public function display()
         $configurator = new Configurator();
         $this->ss->assign('EMAIL_ENABLE_AUTO_SEND_OPT_IN', ($configurator->config['email_enable_auto_send_opt_in']) ? "checked='checked'" : "");
```

Comment:
PHP Notice:  Undefined index: email_enable_auto_send_opt_in in /var/www/html/SuiteCRM/modules/EmailMan/views/view.config.php on line 139, referer: http://localhost/SuiteCRM/index.php?module=Administration&action=index

Reply:
done